src/app/tests/suites/certification/Test_TC_S_2_2.yaml

```diff
@@ -577,7 +763,7 @@ tests:
       response:
           values:
               - name: "Status"
-                value: 0x00
+                value: 0x00 || 0x89
```

Comment:
Does this actually do the right thing?  I don't think it does....

Reply:
Reverted in latest commit.